opentelemetry-jaeger/src/lib.rs

```diff
@@ -519,6 +500,53 @@ fn links_to_references(
     }
 }
 
+impl Into<jaeger::Span> for trace::SpanData {
```

Comment:
Do we still need this implementation? I feel like we may as well remove it otherwise.

Reply:
Yeah make sense. Will remove it.